tests/servers.pm

```diff
@@ -2587,15 +2587,24 @@ sub startservers {
                 if(stopserver('https')) {
                     return ("failed stopping HTTPS server with different cert", 3);
                 }
+                # also stop http server, we do not know which state it is in
```

Comment:
The comments in stopserver() says that it already does this "given a stunnel based ssl server, also kill non-ssl underlying one". Is that not working?